ocaml/idl/datamodel.ml

```diff
@@ -641,6 +641,10 @@ let _ =
     ~doc:"VGPU type is not one of the PGPU's supported types." ();
   error Api_errors.vgpu_type_not_compatible_with_running_type ["pgpu"; "type"; "running_type"]
     ~doc:"VGPU type is not compatible with one or more of the VGPU types currently running on this PGPU" ();
+  error Api_errors.vgpu_destination_incompatible ["reason"; "vgpu"; "host"]
+    ~doc:"The VGPU is not compatible with any of the destination's PGPU." ();
```

Comment:
Text should end "...destination's PGPU**s**."
or could be
"The VGU is not compatible with any PGPU in the destination."